nextflow.config

```diff
@@ -254,6 +254,12 @@ env {
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
+// Set default registry for Docker and Podman independent of -profile
+// Will not be used unless Docker / Podman are enabled
+// Set to your registry if you have a mirror of containers
+docker.registry = 'quay.io'
+podman.registry = 'quay.io'
+
```

Comment:
Isn't this in the docker and podman profiles?

Reply:
good catch...i missed them somehow

Reply:
got ya...i remember the post on slack now...will update thanks @adamrtalbot